test-suite/tools/update-compat/run.sh

```diff
@@ -6,4 +6,4 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 # we assume that the script lives in test-suite/tools/update-compat/,
 # and that update-compat.py lives in dev/tools/
 cd "${SCRIPT_DIR}/../../.."
-dev/tools/update-compat.py --assert-unchanged --master || exit $?
```

Comment:
Is this change desired?

Reply:
Yes, this is automatically changed by the compat scripts so that the test-suite can check that things are like they should be.